diff -uNr cryopid-0.4.orig/cpimage.h cryopid-0.4/cpimage.h
--- cryopid-0.4.orig/cpimage.h	2005-04-25 01:13:01.000000000 +0200
+++ cryopid-0.4/cpimage.h	2005-06-24 01:24:34.000000000 +0200
@@ -4,7 +4,7 @@
 #include <sys/socket.h>
 #include <linux/types.h>
 #include <netinet/in.h>
-#include <linux/un.h>
+#include <sys/un.h>
 #include <linux/user.h>
 #include <linux/unistd.h>
 #include <asm/ldt.h>
diff -uNr cryopid-0.4.orig/cryopid.h cryopid-0.4/cryopid.h
--- cryopid-0.4.orig/cryopid.h	2005-04-25 01:13:01.000000000 +0200
+++ cryopid-0.4/cryopid.h	2005-06-24 01:26:41.000000000 +0200
@@ -33,13 +33,13 @@
 void *xmalloc(int len);
 
 /* writer_raw.c */
-extern struct stream_ops raw_ops;
+// extern struct stream_ops raw_ops;
 
 /* writer_buffered.c */
-extern struct stream_ops buf_ops;
+// extern struct stream_ops buf_ops;
 
 /* writer_lzo.c */
-extern struct stream_ops lzo_ops;
+// extern struct stream_ops lzo_ops;
 
 /* process.c */
 int do_syscall(pid_t pid, struct user_regs_struct *regs);
